src/Classes/Particle/Repulser.ts

```diff
@@ -24,14 +25,48 @@
         const mouseMoveStatus = container.interactivity.status === "mousemove";
         const hoverMode = options.interactivity.events.onhover.mode;
         const clickMode = options.interactivity.events.onclick.mode;
+        const divMode = options.interactivity.events.ondiv.mode;
 
         if (mouseMoveStatus && hoverEnabled && Utils.isInArray(HoverMode.repulse, hoverMode)) {
             this.hoverRepulse();
         } else if (clickEnabled && Utils.isInArray(ClickMode.repulse, clickMode)) {
             this.clickRepulse();
+        } else if (options.interactivity.events.ondiv.enable && Utils.isInArray(DivMode.repulse, divMode)) {
+            this.divRepulse();
         }
     }
 
+    private divRepulse(): void {
+        const container = this.container;
+        const options = container.options;
+        const particle = this.particle;
+
+        const elem = document.getElementById(options.interactivity.events.ondiv.el) as HTMLElement;
+        const pos = {
+            x: (elem.offsetLeft + elem.offsetWidth / 2),
+            y: (elem.offsetTop + elem.offsetHeight / 2),
+        };
+        let divWidth = elem.offsetWidth / 2;
+
+        if (container.retina.isRetina) {
+            pos.x *= container.canvas.pxRatio;
+            pos.y *= container.canvas.pxRatio;
+            divWidth *= container.canvas.pxRatio
+        }
+
+        const dx_div = particle.position.x - pos.x;
+        const dy_div = particle.position.y - pos.y;
+        const dist_div = Math.sqrt(dx_div * dx_div + dy_div * dy_div);
+
+        const normVec = {x: dx_div / dist_div, y: dy_div / dist_div};
+        const repulseRadius = divWidth;
+        const velocity = 100;
+        const repulseFactor = Utils.clamp((1 / repulseRadius) * (-1 * Math.pow(dist_div / repulseRadius, 4) + 1) * repulseRadius * velocity, 0, 50);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)